scripts/run_smoke_test.sh

```diff
@@ -47,8 +47,8 @@ run_smoke_test() {
     local waitcmd="phase=\"\"; echo \"Waiting for test results from pod\"; until [[ \$phase = Succeeded ]]; do sleep 1; phase=\$(kubectl get pod test -ojsonpath=\"{.status.phase}\"); if [[ \$phase = Failed ]]; then echo \$phase; kubectl get pods -A; exit 1; fi; echo \$phase; done"
     kubectl run nginx --image=nginx --port=80
     kubectl expose pod nginx
-    kubectl run test --restart=Never --image=radial/busyboxplus:curl --command -- /bin/sh -c "$curlcmd"
-    timeout 180s bash -c "$waitcmd"
+    kubectl run test --restart=Never --image=elotl/debug --command -- /bin/sh -c "$curlcmd"
+    timeout 300s bash -c "$waitcmd"
```

Comment:
Yikes, was this regularly timing out?  Instances should start up in < 2 minutes.

Reply:
> Yikes, was this regularly timing out? Instances should start up in < 2 minutes.

There's a bit of an extra lag here, since the timeout starts ticking almost immediately after the cluster is up. So Kip needs to be installed, images get pulled, the Kip pod comes up, does the usual initialization, etc. 180s is usually enough, but occasionally I've seen timeouts where it expired right when the test pods were about to start running.